pkg/document/document.go

```diff
@@ -200,10 +200,6 @@ func (d *document) processLoad(dep Document, load LoadStatement) {
 	for _, ls := range load.Symbols {
 		if sym, found := symMap[ls.Name]; found {
 			sym.Name = ls.Alias
-			sym.Location = protocol.Location{
```

Comment:
Ah, at first I wondered why this went away but I figured it out, to preserve the original location.

I'm realizing now though that we are going to need to have a way to resolve extension URIs into filenames if we want to be able to jump into them.

Reply:
If we go through the tilt reconciler, we can use [Extension.Status.Path](https://api.tilt.dev/core/extension-v1alpha1#ExtensionStatus)